site/content/installation/ngf-images/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "NGINX Gateway Fabric Images"
```

Comment:
If we're going to have a directory dedicated to NGF images, I think we should move `building-the-images.md` under this directory.

Reply:
will do !